### ISSUE TITLE
Spec m.room.bot.options

### ISSUE THREAD

**Issue by KitsuneRal** [matrix-org/matrix-doc#1409]
At least on a generic level like 'This event allows to configure bots participating in the room; configuration options are bot-specific'.

**Comment by Half-Shot** [comment]
Sounds quite similar to @turt2live's #1206

**Comment by KitsuneRal** [comment]
Yes but this is already a thing - as a client author, I wouldn't mind visualising it in a sensible way. Perfectly fine to instead just wait for #1206 to land.
